elasticdl_preprocessing/utils/analyzer_utils.py

```diff
@@ -5,6 +5,16 @@
 import os
 
 
+class AnalysisEnvTemplate:
```

Comment:
Put this in a separate file? Also is this only used internally? If so, you can start the name with underscore.

Reply:
Yes, it is better to put it into a separate file.